### DISCUSSION TITLE
Upscaling Tilesets

### DISCUSSION THREAD

**Discussion by FatalFrog** [I-am-Erk/CDDA-Tilesets#1095] [Q&A]
I am rather proficient at image editing, drawing, and just modding in general, so this is not a "how do I" post and rather a "can I" post

My question is, would it be possible for me to upscale this (for personal use) to 64x64 or would Cataclysm not be able to handle it.

**Comment by secretstamos** [comment]
@FatalFrog Utica has some 96x96 sized icons. I don't think anyone's attempted a full tileset beyond 32x32 however.
